### DISCUSSION TITLE
Error when creating the starter app

### DISCUSSION THREAD

**Discussion by ChristianGreinke** [vercel/next.js#16185] [Help]
Hi All, 

when running the command:

npx create-next-app nextjstest --use-npm --example "https://github.com/vercel/next-learn-starter/tree/master/learn-starter"

it returns with the following error:

```
npx: installed 1 in 2.605s
Unexpected token *
```

I can't proceed from there

```
node: v8.10.0
npm: 3.5.2
npx: 10.2.0
```

Thanks for your help

**Comment by timneutkens** [comment]
You will want to update your version of Node.js, Node 8 is end of life: https://github.com/nodejs/Release

**Reply by ChristianGreinke** [reply]
Thanks for your insight. Most appreciated. Should have thought of it my self.
